server/views/reports/show.njk

```diff
@@ -167,3 +172,11 @@
     </div>
   </div>
 {% endblock content %}
+
+{% block bodyEnd %}
+  {{ super() }}
+  <script src="/assets/accessible-autocomplete.min.js" type="text/javascript"></script>
+  <script nonce={{ cspNonce }}>
+    accessibleAutocomplete.enhanceSelectElement({defaultValue: 'Find prison', placeHolder: 'Find prison', showAllValues: true, selectElement: document.querySelector('#location')});
```

Comment:
A `.js-` prefixed class preferred to target elements, rather than the ID.  Couple of examples in there such as `.js-repeater-section` and `.js-back-link`. Helps with identifying that it has a purpose, just incase anyone in the future decided to change the id of the select and then didn't update it here.